pallets/anchors/src/lib.rs

```diff
@@ -419,21 +476,16 @@ impl<T: Trait> Module<T> {
 
     fn store_anchor(anchor_id: T::Hash, prefixed_key: &Vec<u8>, stored_until_date_from_epoch: u32, anchor_data_encoded: &[u8]) {
         let child_info = common::generate_child_storage_key(prefixed_key);
-
         child::put_raw(&child_info, anchor_id.as_ref(), &anchor_data_encoded);
 
         // update indexes
         <AnchorEvictDates<T>>::insert(&anchor_id, &stored_until_date_from_epoch);
-        let idx = LatestAnchorIndex::get() + 1;
+        let idx = <LatestAnchorIndex<T>>::get().unwrap_or_default() + 1;
```

Comment:
`checked_add`, we cann then always throw this [error](https://github.com/paritytech/substrate/blob/90cfb952f2e11bc6d327faf04b30dc5f4cf89df8/primitives/runtime/src/lib.rs#L568-L578)